src/components/ColorPicker/components/ColorPickerContent/ColorPickerColorsGrid.jsx

```diff
@@ -22,11 +22,33 @@ export const ColorPickerColorsGrid = React.forwardRef(
       SelectedIndicatorIcon,
       colorSize,
       tooltipContentByColor,
-      colorShape
+      colorShape,
+      showCustomColorPicker,
+      initialCustomColor
     },
     ref
   ) => {
+    const [customColor, setCustomColor] = useState(initialCustomColor);
+    const [showColorPicker, setShowColorPicker] = useState();
```

Comment:
consider being more explicit here
```suggestion
    const [showColorPicker, setShowColorPicker] = useState(false);
```

and same comment on lines 46-47 (`setShowColorPicker()`...)